### DISCUSSION TITLE
I cant hack the ESCAM PT202 camera

### DISCUSSION THREAD

**Discussion by Raynoxis** [roleoroleo/yi-hack-Allwinner-v2#624] [Q&A]
hello,
I bought an escam pt202 camera in chinese version. I started it with the Chinese iphone application and connected to my wifi network. 

It works well with Yi app. 

But when I put the SD card formatted in FAT32 with the contents of the b091qp_0.2.6 archive, the camera seems to start in a loop. It doesn't seem to go as far as the login step. 

The firmware version of the camera given by the Yi app gives: 6.0.02.10_202204131123
S/N on the camera : 202209

Thank you

**Comment by Raynoxis** [comment]
![93060BFB-BB0D-47C2-94C1-D95BC742C176](https://user-images.githubusercontent.com/34026291/236146745-5c643413-09c0-44dd-a489-927025620835.jpeg)

**Reply by Raynoxis** [reply]
I think the hardware changed on the escam pt202, do you know this model ?

**Reply by roleoroleo** [reply]
It's a different hardware platform.
Sorry but this hack is not compatible.

**Reply by Raynoxis** [reply]
Arg , do you know a camera model for outdoor with PTZ which works with your hack ? It seems the PT202 had been updated to an ANYKA processor... thank you so,much

**Reply by roleoroleo** [reply]
Check the last 6 models in the list:
https://github.com/roleoroleo/yi-hack-Allwinner-v2

**Reply by Rushmed** [reply]
Hi.
On my ESCAM PT202 I dont see the Anyka processor but the flashing process ist stuck on boot with floodlight staying on.
Ist this another new unhackable Hardware version?
![20230530_133334](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/59259796/0dbaa370-a0e3-4732-9b13-c25e864b2c6b)

**Reply by roleoroleo** [reply]
Could you post a picture of the other side?

**Reply by Rushmed** [reply]
Here are pics of the backside:
![image](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/59259796/725e6f1d-e9a7-46f9-903c-36c1a0a539c2)
![image](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/59259796/2aa4ad7a-579e-430d-96a2-69d8ad22dec8)
![image](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/59259796/87ef2700-921f-4f62-883d-f0789ca2ce45)
![image](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/59259796/c30c0341-3172-4672-b5f8-8702593f6d73)

**Reply by roleoroleo** [reply]
Sorry but this model is not supported.
It's another hardware platform.

**Comment by Rushmed** [comment]
THX.

**Comment by Rushmed** [comment]
Is there any way to get Hack support for this platform yet?
